hilti/src/compiler/cxx/elements.cc

```diff
@@ -546,8 +546,14 @@ cxx::Formatter& cxx::operator<<(cxx::Formatter& f, const cxx::Block& x) {
             else {
                 if ( compact_block && ! b )
                     f << s << ';';
```

Comment:
We need a similar fix here.

Reply:
Good catch. I'll see if we can actually combine those two blocks.